sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+            applications.data ->> ''studentParentNetContribution'' as "Parental contribution self-reported",
+            -- Parent1 Info
+            parent1_users.first_name as "Parent 1 first name",
+            parent1_users.last_name as "Parent 1 last name",
+            parent1.contact_info -> ''address'' ->> ''postalCode'' as "Parent 1''s postal code",
+            parent1.supporting_data ->> ''totalIncome'' as "Parental income (line 15000)",
+            ''--TBD--'' as "Parental net annual income (calculated value)",
+            ''--TBD--'' as "Parental value of assets",
+            parent1.supporting_data ->> ''cppLine30800'' as "Canada pension plan contribution (line 30800)",
+            parent1.supporting_data ->> ''cppLine31000'' as "Canada pension plan contribution (line 31000)",
+            parent1.supporting_data ->> ''eiLine31200'' as "Employment Insurance Premium (Line 31200)",
+            parent1.supporting_data ->> ''totalIncomeTaxLine43500'' as "Income tax paid (Line 43500)",
+            ''--TBD--'' as "Total taxable income (Line 23600) (calculated value)",
+            parent1.supporting_data ->> ''parentalContributions'' as "Parental contribution to student",
+            parent1.supporting_data ->> ''parentsOtherDependants'' as "Parental dependants other than applicant",
+            ''--TBD--'' as "Other dependants - age of dependant",
+            ''--TBD--'' as "Other dependants - attending high school",
+            ''--TBD--'' as "Other dependants - claimed for disability",
+            -- Parent2 Info
+            parent2_users.first_name as "Parent 2 first name",
+            parent2_users.last_name as "Parent 2 last name",
+            parent2.contact_info -> ''address'' ->> ''postalCode'' as "Parent 2''s postal code",
+            parent2.supporting_data ->> ''totalIncome'' as "Parental income (line 15000)",
+            ''--TBD--'' as "Parental net annual income (calculated value)",
+            ''--TBD--'' as "Parental value of assets",
+            parent2.supporting_data ->> ''cppLine30800'' as "Canada pension plan contribution (line 30800)",
+            parent2.supporting_data ->> ''cppLine31000'' as "Canada pension plan contribution (line 31000)",
+            parent2.supporting_data ->> ''eiLine31200'' as "Employment Insurance Premium (Line 31200)",
+            parent2.supporting_data ->> ''totalIncomeTaxLine43500'' as "Income tax paid (Line 43500)",
+            ''--TBD--'' as "Total taxable income (Line 23600) (calculated value)",
+            parent2.supporting_data ->> ''parentalContributions'' as "Parental contribution to student",
+            parent2.supporting_data ->> ''parentsOtherDependants'' as "Parental dependants other than applicant",
+            ''--TBD--'' as "Other dependants - age of dependant",
+            ''--TBD--'' as "Other dependants - attending high school",
+            ''--TBD--'' as "Other dependants - claimed for disability",
```

Comment:
just wondering, we do have these values in the application and supporting parent. then why it's `TBD`?
![image](https://github.com/bcgov/SIMS/assets/77353155/d655455d-38f7-4b48-bd22-ad2eafb58888)
![image](https://github.com/bcgov/SIMS/assets/77353155/7208063e-69a2-42ee-b62a-24e02d28a466)

Reply:
@andrepestana-aot as discussed, I will just add the fields that I know, which are `TBD` in this PR. I am not expecting you to address all those comments in this ticket, we can address them in the future ticket.